ansible/configs/ansible-multi-node/files/cloud_providers/ec2_cloud_template.j2

```diff
@@ -0,0 +1,373 @@
+#jinja2: lstrip_blocks: "True"
```

Comment:
@tonykay you may delete this file if not using customized ec2 template just to keep everything aligned with default template.